src/resources/views/corporation/tracking.blade.php

```diff
@@ -13,60 +13,61 @@
+            <th>{{ trans('web::seat.last_location') }}</th>
+            <th>{{ trans('web::seat.joined') }}</th>
+            <th>{{ trans('web::seat.last_login') }}</th>
+          </tr>
         </thead>
         <tbody>
 
-        @foreach($tracking as $character)
+          @foreach($tracking as $character)
 
-          <tr>
-            <td data-order="{{ $character->key_ok }}">
-              @if($character->key_ok == 1)
-                <i class="fa fa-check text-success"></i>
-              @else
-                <i class="fa fa-exclamation-triangle text-danger"></i>
-              @endif
-            </td>
-            <td>
-              <a href="{{ route('character.view.sheet', ['character_id' => $character->character_id]) }}">
-                {!! img('character', $character->character_id, 64, ['class' => 'img-circle eve-icon small-icon']) !!}
-                <span rel="id-to-name">{{ $character->character_id }}</span>
-              </a>
-              @if(!is_null($character->ship_type_id))
-                <i class="pull-right" data-toggle="tooltip"
-                   title="" data-original-title="{{ $character->type->typeName }}">
-                  {!! img('type', $character->ship_type_id, 64, ['class' => 'img-circle eve-icon small-icon']) !!}
-                </i>
-              @endif
-            </td>
-            <td>
-              @if(! is_null($character->location))
-              {{ $character->location->name }}
-              @else
-              Unknown Location
-              @endif
-            </td>
-            <td data-order="{{ $character->start_date }}">
-              <span data-toggle="tooltip"
-                    title="" data-original-title="{{ $character->start_date }}">
-                {{ human_diff($character->start_date) }}
-              </span>
-            </td>
-            <td data-order="{{ $character->logon_date }}">
-              <span data-toggle="tooltip"
-                    title="" data-original-title="{{ $character->logon_date }}">
-                {{ human_diff($character->logon_date) }}
-              </span>
-            </td>
-          </tr>
+            <tr>
+              <td data-order="{{ $character->key_ok }}">
+                @if($character->key_ok == 1)
+                  <i class="fa fa-check text-success"></i>
+                @else
+                  <i class="fa fa-exclamation-triangle text-danger"></i>
+                @endif
+              </td>
+              <td>
+                <a href="{{ route('character.view.sheet', ['character_id' => $character->character_id]) }}">
+                  {!! img('character', $character->character_id, 64, ['class' => 'img-circle eve-icon small-icon']) !!}
+                  <span rel="id-to-name">{{ $character->character_id }}</span>
+                </a>
+              </td>
+              <td>
+                <span class="character-id-to-main-character">{{ $character->character_id }}</span>
```

Comment:
Actually, this is causing a collapse with the current standard `id-to-name` resolving function. In case the existing resolver is call before you own, the main_character will not be rendered properly.

If the standard is using `[rel="id-to-name"]` to trigger the ajax query, it's using `span:contains(character_id)` as replacement anchors.

In order to fix this and meet a bit with html5 good process, we should apply the following change to the existing flow :
 - moving `<span rel="id-to-name">character_id</span>` to `<span rel="id-to-name" data-character-id="character_id">Unknown</span>`
- updating the replacing anchor from `span:contains(character_id)` to `span[rel=id-to-name][data-character-id=character_id`

If you could apply the change and also include the data attribute to your PR; that will be awesome ♥.

https://github.com/eveseat/web/blob/master/src/resources/views/includes/javascript/id-to-name.blade.php#L36-L38